.github/workflows/footprint-tracking.yml

```
@@ -35,6 +26,10 @@ jobs:
       ZEPHYR_SDK_INSTALL_DIR: /opt/toolchains/zephyr-sdk-0.13.2
       CLANG_ROOT_DIR: /usr/lib/llvm-12
       ZEPHYR_TOOLCHAIN_VARIANT: zephyr
+    concurrency:
+      group: ${{ github.workflow }}
+      cancel-in-progress: true
+
```

https://github.com/zephyrproject-rtos/zephyr/blob/7392d890e829edb9461f54562f5718752e69c604/.github/workflows/footprint-tracking.yml#L4-L14

Noting that `footprint-tracking` can run on both schedule and tag pushes, this can cause a tag run to be cancelled; though, the consequence of that is no-op since we do not care about the refs when footprint-tracking for now.

```suggestion
    concurrency:
      group: ${{ github.workflow }}-${{ github.ref }}
      cancel-in-progress: true
```

---

applied suggestion

---

actually, changed to use `run_id`, since `ref` is only set for PRs.